php/whatsapp-with-vonage/src/index.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+require(__DIR__ . '/../vendor/autoload.php');
+require(__DIR__ . '/utils.php');
+
+return function ($context) {
+    throw_if_missing($_ENV, [
+        'VONAGE_API_KEY',
+        'VONAGE_API_SECRET',
+        'VONAGE_API_SIGNATURE_SECRET',
+        'VONAGE_WHATSAPP_NUMBER',
+    ]);
+
+    if ($context->req->method === 'GET') {
+        return $context->res->send(get_static_file('index.html'), 200, [
+            'Content-Type' => 'text/html; charset=utf-8',
+        ]);
+    }
+
+    $authorizationHeader = isset($context->req->headers["authorization"]) ? $context->req->headers["authorization"] : "";
+    $token = explode(" ", $authorizationHeader)[1] ?? "";
+    $jwtParts = explode(".", $token);
+    $payload = base64_decode($jwtParts[1]);
+    $decodedPayload = json_decode($payload, true);
+    
+
+    if(hash("sha256",$context->req->bodyRaw) !== $decodedPayload["payload_hash"]){
+        $context->res->json([
+            'ok' => false,
+            'error' => "Payload Mismatch"
+        ], 400);
+    };
+
+    try {
+    throw_if_missing($context->req->body, ['from','text']);
```

Comment:
It would be good to run a formatter to fix style inconsistencies

Reply:
Fixed